src/checking/issue_mapping/issue_mapping.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * Imports
+ */
```

Comment:
Tl;dr: I don't think this comment here is necessary.

Complete story:
Don't get me wrong; comments are great. But they need to add value to what they are commenting. If they are cluttering more than they are giving to the reader, they should be removed. 
I would avoid comments where possible, since they are cluttering the code and easily go "out of sync" with the actual code (comments are no "part of the code", since they have no semantics themselves). Whenever possible, use "talking" method- and class-names: `function getUndecidedVoters(): Person[]` is an excellent method name that needs no further comment.